docs/en/latest/plugins/openid-connect.md

```diff
@@ -27,58 +27,62 @@ description: OpenID Connect allows the client to obtain user information from th
-| proxy_opts.http_proxy     | string   | False    |                       | http://proxy-server:port         | HTTP proxy server address.                                                                                                                                                                                                            |
-| proxy_opts.https_proxy    | string   | False    |                       | http://proxy-server:port         | HTTPS proxy server address.                                                                                                                                                                                                           |
-| proxy_opts.http_proxy_authorization  | string   | False    |                       | Basic [base64 username:password] | Default `Proxy-Authorization` header value to be used with `http_proxy`. Can be overridden with custom `Proxy-Authorization` request header.                                                                                                                                                              |
-| proxy_opts.https_proxy_authorization | string   | False    |                       | Basic [base64 username:password] | Default `Proxy-Authorization` header value to be used with `https_proxy`. Cannot be overridden with custom `Proxy-Authorization` request header since with with HTTPS the authorization is completed when connecting.                         |
-| proxy_opts.no_proxy                  | string   | False    |                       |                                  | Comma separated list of hosts that should not be proxied.                                                                                                                                                                             |
-| authorization_params                 | object   | False    |                       |                                  | Additional parameters to send in the request to the authorization endpoint.                                                                                                                                                    |
-| client_rsa_private_key | string | False |  |  | Client RSA private key used to sign JWT. |
-| client_rsa_private_key_id | string | False |  |  | Client RSA private key ID used to compute a signed JWT. |
-| client_jwt_assertion_expires_in | integer | False | 60 |  | Life duration of the signed JWT in seconds. |
+| session_contents   | object   | False    |                   |             | Session content configurations. If unconfigured, all data will be stored in the session. |
```

Comment:
About this `session_contents` update from https://github.com/apache/apisix/pull/11892

@bzp2010 I think we should update the plugin schema as well. Could you help confirm updating the schema & associated tests are sufficient? Should these changes be done in this PR or open a new one? Thanks.